src/main/java/mat/client/shared/MatContext.java

```diff
@@ -87,1492 +86,1480 @@
-			currentLibraryInfo.setCqlLibraryVersion(s);
-		}
-	}
-
-	public CQLLibraryServiceAsync getLibraryService() {
-		if(cqlLibraryService == null){
-			cqlLibraryService = (CQLLibraryServiceAsync) GWT.create(CQLLibraryService.class);
-		}
-		return cqlLibraryService;
-	}
-	
-	/**
-	 * Checks if is measure is CQL Measure depending 
-	 * on Measure release version. If the releaseVersion is null, then it is not a CQL measure. 
-	 *
-	 * @param releaseVersion the release version
-	 * @return true, if is CQL measure, false if it is not a cql measure or the release version is null
-	 */
-	public boolean isCQLMeasure(String releaseVersion) {
-		
-		if(releaseVersion == null) {
-			return false; 
-		}
-		
-		String str[] = releaseVersion.replace("v", "").split("\\.");
-		int versionInt = Integer.parseInt(str[0]);
-		if(versionInt<5){
-			return false;
-		}
-		
-		return true;
-	}
-
-	public List<CQLIdentifierObject> getIncludedDefNames() {
-		return includedDefNames;
-	}
-
-	public void setIncludedDefNames(List<CQLIdentifierObject> includedDefNames) {
-		this.includedDefNames = includedDefNames;
-	}
-
-	public List<CQLIdentifierObject> getIncludedFuncNames() {
-		return includedFuncNames;
-	}
-
-	public void setIncludedFuncNames(List<CQLIdentifierObject> includedFuncNames) {
-		this.includedFuncNames = includedFuncNames;
-	}
-
-	public List<CQLIdentifierObject> getIncludedValueSetNames() {
-		return includedValueSetNames;
-	}
-
-	public void setIncludedValueSetNames(List<CQLIdentifierObject> includedValueSetNames) {
-		this.includedValueSetNames = includedValueSetNames;
-	}
-
-	public List<CQLIdentifierObject> getIncludedParamNames() {
-		return includedParamNames;
-	}
-
-	public void setIncludedParamNames(List<CQLIdentifierObject> includedParamNames) {
-		this.includedParamNames = includedParamNames;
-	}
-
-	public List<CQLIdentifierObject> getIncludedCodeNames() {
-		return includedCodeNames;
-	}
-
-	public void setIncludedCodeNames(List<CQLIdentifierObject> includedCodeNames) {
-		this.includedCodeNames = includedCodeNames;
-	}
-	
-	public void setIncludedValues(SaveUpdateCQLResult result) {
-		includedValueSetNames = result.getCqlModel().getCQLIdentifierValueSet();
-		includedCodeNames = result.getCqlModel().getCQLIdentifierCode();
-		includedParamNames = result.getCqlModel().getCQLIdentifierParam();
-		includedDefNames= result.getCqlModel().getCQLIdentifierDefinitions();
-		includedFuncNames = result.getCqlModel().getCQLIdentifierFunctions();
-	}
-
-	public void getCQLConstants() {
-		CQLConstantServiceAsync cqlConstantService = (CQLConstantServiceAsync) GWT.create(CQLConstantService.class);
-
-		cqlConstantService.getAllCQLConstants(new AsyncCallback<CQLConstantContainer>() {
-		
-			@Override
-			public void onFailure(Throwable caught) {
-				Window.alert(MatContext.get().getMessageDelegate().getGenericErrorMessage());
-			}
-
-			@Override
-			public void onSuccess(CQLConstantContainer result) {
-				cqlConstantContainer = result; 
```

Comment:
I'm a bit concerned about removing the cast to BonnieServiceAsync here. I remember it being added for a reason, I don't remember that reason. But especially since we cant test the bonnie connection locally or in dev right now I don't have warm and fuzzys removing this.

Reply:
It's a false positive diff caused with sources re-formatting. Mat files have mixed identiation tab/spaces. Somehow GIT doesn't show the diffs properly. But Intellij shows it correctly. No casting was affected.